test/Find.lean

```diff
@@ -1,5 +1,6 @@
 import Mathlib.Tactic.Find
 
+set_option autoImplicit true
 theorem add_comm_zero : 0 + n = n + 0 := Nat.add_comm _ _
```

Comment:
```suggestion
theorem add_comm_zero {n} : 0 + n = n + 0 := Nat.add_comm _ _
```
seems like the smaller patch here, and also ensure we don't let `#find` somehow rely on `autoImplicit`